Gopkg.toml

```diff
@@ -27,25 +27,62 @@
 ignored = ["github.com/kubeflow/kubebench"]
 
 [[constraint]]
-  name = "github.com/ghodss/yaml"
-  version = "1.0.0"
+  branch = "master"
+  name = "github.com/lib/pq"
 
 [[constraint]]
-  name = "github.com/kubeflow/pytorch-operator"
-  branch = "master"
+  branch = "v2"
+  name = "gopkg.in/yaml.v2"
```

Comment:
changing github.com/ghodss/yaml to gopkg.in/yaml.v2 may cause problems in existing codes. is this necessary in the new codes?